app/config/collections.php

```diff
@@ -1932,9 +1751,9 @@
                 '$id' => '_key_search',
                 'type' => Database::INDEX_FULLTEXT,
                 'attributes' => ['search'],
-                'lengths' => [2048],
-                'orders' => [Database::ORDER_ASC],
-            ]
+                'lengths' => [],
+                'orders' => [],
+            ],
```

Comment:
Can you revert the change to the functions collection. This change shouldn't be a part of this PR

Reply:
what do you mean?